packages/aragon-cli/package.json

```diff
@@ -67,11 +62,10 @@
     "go-platform": "^1.0.0",
     "http-server": "^0.11.1",
     "ignore": "^4.0.6",
-    "inquirer": "^6.4.1",
+    "inquirer": "^7.0.0",
     "internal-ip": "^4.3.0",
     "ipfs-api": "^26.1.2",
     "ipfs-http-client": "^33.0.1",
-    "js-sha3": "^0.8.0",
```

Comment:
I think `js-sha3` is still used at a couple of places.

Reply:
Good catch forgot to update to use `web3.utils`